upgrade/upgrade-scripts/common.sh

```diff
@@ -204,3 +204,10 @@ function set_upgrade_property() {
 	source_upgrade_properties ||
 		die "failed to read properties file after setting '$1=$2'"
 }
+
+function turn_verbose_logging_on() {
+    exec 4> >(logger -t $0)
```

Comment:
Should we be using any specific facility name and/or log level here? i.e. via the `-p` option?